cmd/pro/update_provider.go

```diff
@@ -73,5 +74,10 @@ func (cmd *UpdateProviderCmd) Run(ctx context.Context, args []string) error {
 		return fmt.Errorf("update provider %s: %w", provider.Name, err)
 	}
 
+	err = providercmd.ConfigureProvider(ctx, provider, devPodConfig.DefaultContext, []string{}, false, false, nil, log.Discard)
```

Comment:
nit: I think it would be nicer to move common functionality to `pkg` and not have the commands import each other